packages/react-components/src/templates/ManuscriptForm.tsx

```diff
@@ -703,6 +720,54 @@ const ManuscriptForm: React.FC<ManuscriptFormProps> = ({
                 />
               )}
 
+            {watch('versions.0.requestingApcCoverage') ===
+            'Already submitted' ? (
+              <>
+                <Controller
+                  name="versions.0.submitterName"
+                  control={control}
+                  rules={{
+                    required: "Please enter the submitter's name.",
+                    maxLength: {
+                      value: 256,
+                      message: 'This title cannot exceed 256 characters.',
```

Comment:
Maybe "Submitter's name cannot exceed 256 characters."?

Reply:
Thanks! I'll change that!